environments/base/manifests/query-frontend-cache-service.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app.kubernetes.io/component: store-cache
```

Comment:
Seems this label is hardcoded in the Memcached library. Does it make sense to make it configurable and change it to `query-frontend-cache`?  I am not sure if this affects anything like dashboard queries.

Reply:
This should be easily overridden in this repo by changing common labels.

Reply:
Done. Please take a look. Addressed in the latest commit https://github.com/observatorium/deployments/pull/360/commits/86df82dd27a8c1b44ef9d8dfbe57ca8b039ad46b